sources/packages/web/src/components/aest/students/assessment/Request.vue

```diff
@@ -10,7 +10,7 @@
         >
         </body-header>
         <content-group class="mt-4">
-          <toggle-content :toggled="!requestedAssessment.length">
+          <toggle-content :toggled="requestedAssessment.length === 0">
```

Comment:
`!requestedAssessment.length` is also correct. `!0` is false.

Reply:
Also !requestedAssessment.length is fail safe(Even if API is down it will load the playful cat). I recommend that one strongly.